src/backend/aspen/api/views/phylo_runs.py

```diff
@@ -168,10 +167,7 @@ async def get_serializable_runs(
         joinedload(PhyloRun.user),  # For Pydantic serialization
         joinedload(PhyloRun.group),  # For Pydantic serialization
     )
-    # TODO - DECOVIDIFY - remove the None check!
-    query = query.filter(
-        or_(PhyloRun.pathogen == pathogen, PhyloRun.pathogen_id == None)  # noqa: E711
-    )
+    query = query.filter(PhyloRun.pathogen == pathogen)  # noqa: E711
```

Comment:
are we ready to remove the none check?

Reply:
Yep, we ran the final migration in prod on fri.